services/net/image/ImageAction.cs

```diff
@@ -142,6 +142,8 @@ public ImageAction(IKafkaMessenger producer, IApiService api, IOptions<ImageOpti
         else
         {
             this.Logger.LogError("SSH Private key file does not exist");
```

Comment:
Don't do three separate logs.  Make it a single statement if there is a failure.  Also don't use `LogInformation` for errors.